engine/src/test/java/io/zeebe/engine/processing/deployment/DeploymentCreateProcessorTest.java

```diff
@@ -73,12 +74,12 @@ public void shouldRejectTwoCreatingCommands() {
             DeploymentIntent.CREATE,
             DeploymentIntent.CREATED,
             DeploymentIntent.CREATE,
-            DeploymentIntent.CREATE);
+            DeploymentIntent.CREATED);
     //
     Assertions.assertThat(collect)
         .extracting(Record::getRecordType)
         .containsExactly(
-            RecordType.COMMAND, RecordType.EVENT, RecordType.COMMAND, RecordType.COMMAND_REJECTION);
+            RecordType.COMMAND, RecordType.EVENT, RecordType.COMMAND, RecordType.EVENT);
```

Comment:
Since the behavior changes, we should also change the name of the test method `shouldRejectTwoCreatingCommands()`.